### PR TITLE
Add service template for azimutt

### PR DESCRIPTION
**Pull request by tonylyjones**
Service template for [Azimutt](https://azimutt.app/) - [Repo here](https://github.com/azimuttapp/azimutt)

> Next-Gen ERD: Design, Explore, Document and Analyze your database.

### CODE REVIEW

File: templates/compose/azimutt.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# documentation: https://docs.azimutt.app/
+# slogan: Next-Gen ERD: Design, Explore, Document and Analyze your database.
+# tags: erd, entity-relationship diagram, database tool, database schema, diagram
+# icon: svgs/azimutt.svg
+# port: 4000
+
+services:
+  database:
```
Comment (loicknuchel) [inline templates/compose/azimutt.yaml:8]: Will need a S3 service as well

File: templates/compose/azimutt.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    image: postgres
+    restart: always
+    container_name: azimutt-db
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=$SERVICE_PASSWORD_POSTGRESQL
+      - POSTGRES_USER=$SERVICE_USER_POSTGRESQL
+      - POSTGRES_DB=azimutt_dev
```
Comment (loicknuchel) [inline templates/compose/azimutt.yaml:17]: If you choose the name, maybe `azimutt_app` is better than `azimutt_dev` (that I use for dev env ^^)

**Review by loicknuchel — APPROVED** [review]

**Comment by loicknuchel** [comment]
Hi @andrasbacsai,
Any feedback on this?
Cheers!

**Comment by telenc** [comment]
Capture d’écran 2024-08-06 à 22 18 50

@tonylyjones i got this error on postgres

**Comment by loicknuchel** [comment]
Hi @telenc 
Thank you for your message, I'm Loïc, the owner of Azimutt.
The gateway is not deployed with the backend, and expected to be launched on the user desktop to keep the db connection private from the server (and also access local databases).
I see you are using a "Custom gateway", what is it?
Have you deployed https://github.com/azimuttapp/azimutt/tree/main/gateway somewhere?
Have you tried with the `npx azimutt@latest gateway` command? (it launches the gateway on your desktop to be used by Azimutt).
Let me know how it goes.
Cheers!

**Comment by peaklabs-dev** [comment]
Thank you for the PR 💜. I will test and fix it if needed.

**Pull request merged**